packages/playground/windows/playground/HostingPane.xaml.cpp

```diff
@@ -419,19 +419,20 @@ void HostingPane::StoreFilenameSettings() {
 void HostingPane::InitComboBoxes() {
   m_jsFileNames = ref new Platform::Collections::Vector<String ^>();
 
-  m_jsFileNames->Append(L"Samples\\rntester");
   m_jsFileNames->Append(L"Samples\\accessible");
-  m_jsFileNames->Append(L"Samples\\callbackTest");
+  m_jsFileNames->Append(L"Samples\\animation");
   m_jsFileNames->Append(L"Samples\\calculator");
+  m_jsFileNames->Append(L"Samples\\callbackTest");
   m_jsFileNames->Append(L"Samples\\click");
-  m_jsFileNames->Append(L"Samples\\customViewManager");
   m_jsFileNames->Append(L"Samples\\control");
   m_jsFileNames->Append(L"Samples\\flexbox");
   m_jsFileNames->Append(L"Samples\\focusTest");
   m_jsFileNames->Append(L"Samples\\geosample");
   m_jsFileNames->Append(L"Samples\\image");
   m_jsFileNames->Append(L"Samples\\index");
+  m_jsFileNames->Append(L"Samples\\messages");
   m_jsFileNames->Append(L"Samples\\mouse");
+  m_jsFileNames->Append(L"Samples\\rntester");
```

Comment:
> m_jsFileNames->Append(L"Samples\\rntester"); [](start = 1, length = 45)

Does this change mean that RNTester will no longer be selected by default?  I'd prefer to keep RNTester selected by default as RNTester is the primary reason for using Playground.

Reply:
It's still selected by default, it's just in alphabetical order now.